packages/typescript/src/lib/generateTypesStats.ts

```diff
@@ -6,8 +6,9 @@ export const generateTypesStats = (
   normalizeOptions: NormalizeOptions
 ) => {
   return Object.entries(filesMap).reduce((acc, [path, contents]) => {
-    const filename = path.slice(`${normalizeOptions.distDir}/`.length);
-
+    const filename = path.slice(
```

Comment:
@dmcgoughpax8 can you provide an example what exactly is happening here?

Reply:
@pavandv, yep, so we have a **REMOTE,** using 2.1.3, with a federationConfig that `exposes` a utils library
```
...
exposes: {
  './Exposed': './src/utils/index.ts
}
...
```
and after the plugin has run, has a `/dist/@mf-types/` directory like this
```
  /_types
  Exposed.d.ts
```
which is expected.

Now, we have our **HOST,** using 2.1.3, with a federationConfig similar to
```
...
remotes: {
  'utilities': 'http://localhost:3000/remoteEntry.js'
}
...
```
and after the plugin has run, has a `@mf-types` directory like this.
```
/utilities
  /_types
```
So, the problem here is that the **HOST** isn't fetching `Exposed.d.ts`.

Why?
* `Exposed.d.ts` isn't defined on the returned `TypesStatsJson` fetched here, https://github.com/module-federation/universe/blob/main/packages/typescript/src/plugins/FederatedTypesPlugin.ts#L161

So, my changes include:
* in `TypescriptCompiler.createHost`, setting `normalizedExposedDestFilePath` as a prop on `TypescriptCompiler.tsDefinitionFilesObj` with `reexport` as its value, but `normalizedExposedDestFilePath` is not formatted the same as the other keys on the `tsDefinitionFilesObj`, so this needs to be accounted for somewhere, else the **HOST** will not build because of the 404 returned from trying to fetch the file at `normalizedExposedDestFilePath`
  * `normalizedExposedDestFilePath` format: `/Users/TheUser/my-module-federation-project/dist/@mf-types/Exposed.d.ts`
  * expected format: `dist/@mf-types/Exposed.d.ts`
* in `generateTypesStats.ts`, updating the index param of the `.slice` call to account for varying path formats

Hope this helps.

Reply:
Apologies for late reply.. Somehow I'm not getting any notifications for the events. 

This sounds good to me 👍